README.md

```diff
@@ -102,3 +102,52 @@ const InstanceLogger = init(({ name }) => {
 
 const george = InstanceLogger({ name: 'George' }); // Created new instance: George
 ```
+
+
+### overrides()
```

Comment:
Frankly I had also bit trouble understanding what is it suppose to do till I found that article. Calling it `overrides` might be misleading because you always compose another initializer that would override instance values with something else. Seeing `overrides` I would assume that it has a priority over anything else. Instead of a _fancy_ name I would for something like `initialAssign`. I see that @ericelliott was against name `assignAllOptions` in a referenced comment, but that's definitely more strange.